<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="5.125">
  <properties />
            <testcase name="loginAsBankManager" time="5.125" classname="testcases.LogInTest">
                <failure type="java.lang.AssertionError"
                  message="it is failed"
              >
<![CDATA[java.lang.AssertionError: it is failed
  org.testng.Assert.fail(Assert.java:94)
  testcases.LogInTest.loginAsBankManager(LogInTest.java:29)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:746)
  org.testng.TestRunner.run(TestRunner.java:600)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
  org.testng.TestNG.runSuites(TestNG.java:1104)
  org.testng.TestNG.run(TestNG.java:1076)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
